app/modals/upsert-stacks-node-api/upsert-stacks-node-api.tsx

```diff
@@ -2,16 +2,16 @@ import React, { FC, useEffect, useRef, useState } from 'react';
 import { useHotkeys } from 'react-hotkeys-hook';
 import { useFormik } from 'formik';
 import { Modal, ButtonGroup, Button, Box, Text, Input } from '@blockstack/ui';
-import { v4 as uuid } from 'uuid';
 import * as yup from 'yup';
 
 import { StacksNode } from '@store/stacks-node';
-import { TxModalHeader, TxModalFooter } from '../transaction/transaction-modal-layout';
 import { ErrorLabel } from '@components/error-label';
 import { ErrorText } from '@components/error-text';
 import { capitalize } from '@utils/capitalize';
-import { Api } from '../../api/api';
+import { Api } from '@api/api';
 import { safeAwait } from '@utils/safe-await';
+import { generateRandomHexString } from '@crypto/key-generation';
+import { TxModalHeader, TxModalFooter } from '../transaction/transaction-modal-layout';
```

Comment:
should be absolute? or too long?

Reply:
This component specifically v soon to be refactored!